src/Constants.h

```diff
@@ -107,4 +107,21 @@ const int MAX_NUM_CFR_CHANGE_TIMES = 100; /**< To allow IFR to scale over time.
 
 #define _I64(x) static_cast<int64_t>(x)
 
+// Settings (numbers not arbitrary - don't change without checking)
+const int PrimarySchool		= 0; 
+const int SecondarySchool	= 1; 
+const int University		= 2; 
+const int Workplace			= 3; 
+const int House				= 4;
```

Comment:
const int House = NUM_PLACE_TYPES;

const int Spatial = NUM_PLACE_TYPES + 1;

Reply:
Don't think this will work, would work if const int House = P.NumPlaceTypes and const int Spatial = P.NumPlaceTypes +1; but don't think that will work with const

Reply:
The above will work, but will waste a tiny bit of memory if P.NumPlaceTypes turns out less than NUM_PLACE_TYPES. Alternative is to declare it as non-const, and then set it once you know what P.NumPlaceTypes is - but I'm not sure that is worth the bother in this case.